lib/screens/home/pages/requests/requests_page.dart

```diff
@@ -132,3 +119,17 @@ class _RequestsPageState extends State<RequestsPage> {
     );
   }
 }
+
+class NoRequestHelper extends StatelessWidget {
```

Comment:
Great job. 
I'd change `NoRequestHelper` -> `NoRequestsInfo` or `NoRequestsText` and we're good to go (because it's not really a *helper*)! 
🎉💥

Reply:
Done. :)